lib/mix/lib/mix/dep/loader.ex

```diff
@@ -304,8 +304,8 @@ defmodule Mix.Dep.Loader do
     |> elem(0)
   end
 
-  defp rebar_children(root_config, manager) do
-    from = Path.absname("rebar.config")
+  defp rebar_children(root_config, manager, dest) do
+    from = Path.join(dest, "rebar.config") |> Path.absname
```

Comment:
It's ugly to pipe `Path.join` because both arguments are the subject. I don't think piping makes sense in those cases. I'd rather have `Path.join(dest, "rebar.config") |> Path.absname` or `Path.absname(Path.join(dest, "rebar.config"))` than `dest | Path.join("rebar.config") |> Path.absname`.

I will change to no pipes if we don't single pipes here. Please let me know.